src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -178,10 +178,7 @@ private static IEnumerable<Signer> DeserializeSigners(BinaryReader reader, int m
             for (int i = 0; i < count; i++)
             {
                 Signer signer = reader.ReadSerializable<Signer>();
-                if (i > 0 && signer.Scopes == WitnessScope.FeeOnly)
```

Comment:
`FeeOnly` in different positions?

Reply:
Maybe we should rename it to `None`.

Reply:
If we add in the comment that it can be used for fee payment, `None` it's good for me

Reply:
Done.